server/features/events_spike.feature

```diff
@@ -96,6 +96,10 @@ Feature: Events Spike
         }
         """
         Then we get OK response
+        When we post to "/events/#events._id#/lock"
```

Comment:
Spike action's shouldn't require a lock (originally they didn't, check the service [here](https://github.com/superdesk/superdesk-planning/blob/develop/server/planning/events/events_spike.py#L45) and [here](https://github.com/superdesk/superdesk-planning/blob/develop/server/planning/events/events_spike.py#L251)).

So I'm guessing [here](https://github.com/superdesk/superdesk-planning/blob/async/server/planning/events/events_spike.py#L247) and [here](https://github.com/superdesk/superdesk-planning/blob/async/server/planning/events/events_spike.py#L297) we would send `require_lock = False`

Reply:
I guess the same goes for any other item service that had the `REQUIRE_LOCK = False` on the old eve service, we would need to update the new endpoint/utils to not require lock either